normandy/settings.py

```diff
@@ -116,6 +124,11 @@ class Production(Base):
     """Settings for the production environment."""
     STATICFILES_STORAGE = 'whitenoise.django.GzipManifestStaticFilesStorage'
 
+    DEFAULT_FILE_STORAGE = 'storages.backends.s3boto.S3BotoStorage'
+    AWS_ACCESS_KEY_ID = values.SecretValue()
```

Comment:
`SecretValue` throws an error if it doesn't have a configuration value, right? Is that what we want?